core/update.php

```diff
@@ -228,7 +228,7 @@ function update_results_page() {
   }
 
   if (settings_get('update_free_access')) {
-    $output .= "<p><strong>Reminder: Don't forget to set the <code>\$settings[&#39;update_free_access&#39;]</code> value in your <code>settings.php</code> file back to <code>FALSE</code>.</strong></p>";
+    backdrop_set_message(t("Reminder: Don't forget to set the <code>\$settings['update_free_access']</code> value in your <code>settings.php</code> file back to <code>FALSE</code>."), 'warning');
```

Comment:
I wonder if it makes sense here to add the code snippets as parameters to t().

`$setting = '<code>$settings[\'update_free_access\']</code>';`

`t('Reminder ... !setting ... !filename')`; (Placeholders with `!` are inserted as-is.

That would move all markup out of the translatable string. And neither setting nor filename need translation.
Totally optional, though...

Reply:
This function is called as a case in [the switch statement below](https://github.com/backdrop/backdrop/blob/b2690394c575ffe659186a6a818dba93da418f6e/core/update.php#L538), and it seems neither this nor any of the other functions called return any translated code (there is no `t()` in either of them). I cannot say why, but for consistency we perhaps shouldnt be using `t()` here at all?

Reply:
@indigoxela, @docwilmot I've now removed `t()` and file this as a follow-up, to figure out why strings are not being translated in update.php: https://github.com/backdrop/backdrop-issues/issues/5394